help_variables.json

```diff
@@ -5178,6 +5178,21 @@
       "group-id": "36",
       "type": "boolean"
     },
+    "gl_part_bloodtrails": {
+      "default": "0",
```

Comment:
It's a bit annoying that the cvar declarations aren't generated from the json files. Would have been neat. Update to 1 as you have in the source.

Reply:
Fixed. So much for my attempts at squashing commits...